base/shell.jl

```diff
@@ -81,7 +81,7 @@ function shell_parse(raw::AbstractString, interp::Bool)
                     if done(s,k)
                         error("unterminated double quote")
                     end
-                    if s[k] == '"' || s[k] == '$'
+                    if s[k] == '"' || s[k] == '$' || s[k] == '\\'
```

Comment:
If I type this into a current Julia, I receive an error instead of the last line. (You asked for the "expected" output.) The test cases I added to `spawn.jl` also report a disagreement where parsing yields too many backslashes -- sorry, I don't have the source string handy, it was more complex.